addons/api/mirage/config.js

```diff
@@ -1,20 +1,35 @@
-import config from '../config/environment';
+import { discoverEmberDataModels } from "ember-cli-mirage";
+import { createServer } from 'miragejs';
+import EnvironmentConfig from '../config/environment';
```

Comment:
I changed the name here to avoid collisions with the `config` variable that Mirage provides (see line 16). Happy to discuss other names or different approach, I found this the more pragmatic to follow.

Reply:
That is a very good point. I do not like variables without context like `config`. In this case was totally fine, because was obvius. But since now we have 2 config variables I love your idea.
- I will rename the mirage config varible to `mirageConfig`.
- I will rename the environment config variable to `environmentConfig`. The Upper camel case was a mistake